scripts/patch-utils.sh

```diff
@@ -135,7 +135,7 @@ function try_module_insert {
 	else
 		# Everything went OK, delete backup
 		printf "\e[32m succeeded\n\e[0m"
-		sudo rm ${tgt_ko}.bckup
+		sudo rm -f ${tgt_ko}.bckup
```

Comment:
Thanks for the amendment, we will accept it.
I just need to understand how did it happen that the backup was not present, as it might indicate an additional issue in the script's flow.
Can you elaborate?

Reply:
As mentioned by RealSense-Customer-Engineering at https://github.com/IntelRealSense/librealsense/issues/840, I used patch-realsense-ubuntu-xenial-jetson-tx2.sh which runs patch-utils; at '#Load the newly-built modules' try_module_insert (at rm command) was unsuccessful in finding any .bkup files.

Reply:
I see, please add inline comment that this flag is for Jetson-tx2 platform scripts.

Reply:
Also, please resubmit the PR to `development` branch  - Travis configuration on `master` branch is currently not synced.